src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/UseSearchValuesTests.cs

```diff
@@ -0,0 +1,1023 @@
+                    private ReadOnlySpan<char> ShortReadOnlySpanOfCharRVAProperty => new[] { 'a', 'e', 'i', 'o', 'u' };
+                    private ReadOnlySpan<char> LongReadOnlySpanOfCharRVAProperty => new[] { 'a', 'e', 'i', 'o', 'u', 'A' };
+                    private ReadOnlySpan<byte> ShortReadOnlySpanOfByteRVAProperty => new[] { (byte)'a', (byte)'e', (byte)'i', (byte)'o', (byte)'u' };
+                    private ReadOnlySpan<byte> LongReadOnlySpanOfByteRVAProperty => new[] { (byte)'a', (byte)'e', (byte)'i', (byte)'o', (byte)'u', (byte)'A' };
+                    private ReadOnlySpan<char> ShortReadOnlySpanOfCharFromToCharArrayProperty => "aeiou".ToCharArray();
+                    private ReadOnlySpan<char> LongReadOnlySpanOfCharFromToCharArrayProperty => "aeiouA".ToCharArray();
+                    private ReadOnlySpan<char> ShortReadOnlySpanOfCharFromConstToCharArrayProperty => ShortConstStringTypeMember.ToCharArray();
+                    private ReadOnlySpan<char> LongReadOnlySpanOfCharFromConstToCharArrayProperty => LongConstStringTypeMember.ToCharArray();
+                    private ReadOnlySpan<char> LongReadOnlySpanOfCharRVAPropertyWithGetAccessor1
+                    {
+                        get => new[] { 'a', 'e', 'i', 'o', 'u', 'A' };
+                    }
+                    private ReadOnlySpan<char> LongReadOnlySpanOfCharRVAPropertyWithGetAccessor2
+                    {
+                        get
+                        {
+                            return new[] { 'a', 'e', 'i', 'o', 'u', 'A' };
+                        }
+                    }
+                    private ReadOnlySpan<char> LongReadOnlySpanOfCharRVAPropertyWithGetAccessor3
+                    {
+                        get
+                        {
+                            Console.WriteLine("foo");
+                            return new[] { 'a', 'e', 'i', 'o', 'u', 'A' };
+                        }
+                    }
+
+                    public Test()
+                    {
+                        LongReadonlyCharArrayFieldWithSimpleFieldModification = new[] { 'a' };
+                        Console.WriteLine(InstanceReadonlyCharArrayField);
+                    }
+
+                    public void Dummy()
+                    {
+                        LongReadonlyCharArrayFieldWithSimpleElementModification[0] = 'a';
+                        Console.WriteLine(InstanceReadonlyCharArrayField[0]);
+                    }
+
+                    private void TestMethod(string str, ReadOnlySpan<char> chars, ReadOnlySpan<byte> bytes)
+                    {
+                        const string ShortConstStringLocal = "foo";
+                        const string LongConstStringLocal = "aeiouA";
+                        string NonConstStringLocal = "aeiouA";
+
+                        _ = chars.IndexOfAny("aeiou");
```

Comment:
See
https://github.com/dotnet/roslyn-analyzers/blob/38ffb6da8ca3906dc7caea18a51ae2c3deaba2e5/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/UseSearchValues.cs#L150-L152

`IndexOfAny` does have specialized implementations and can be an efficient choice as long as you have up to 5 values.
I've intentionally avoided flagging uses that are already getting optimal performance, and many or these tests are checking that distinction between short/long inputs.

Reply:
Yeah, I saw that later, and added another comment "Never mind" here, somehow that comment is not added. 

Though this better to be documented so that people do not file a bug for this